docs/source/noise/column_noise.rst

```diff
@@ -19,6 +19,30 @@ Misreport age
 Write the wrong digits
 ----------------------
 
+Sometimes people may write the wrong number for numeric data such as a street
+number, date, or social security number. This could be intentional or
+accidental. pseudopeople simulates this type of noise in numeric columns by
+randomly selecting some digits from the true number and replacing each selected
+digit with a diferent digit selected uniformly at random.
```

Comment:
```suggestion
randomly replacing some digits with a different digit selected uniformly at random.
```

This seems like enough detail? Or do you think this is missing something important?

Reply:
Nope, sounds good. Thanks!